smartsim/settings/mpirunSettings.py

```diff
@@ -134,6 +137,61 @@ def set_hostlist(self, host_list):
             raise TypeError("host_list argument must be list of strings")
         self.run_args["host"] = ",".join(host_list)
 
+    def set_verbose_launch(self, verbose):
+        """Set the job to run in verbose mode
+
+        This sets ``--verbose``
+
+        :param verbose: Whether the job should be run verbosely
+        :type verbose: bool
+        """
+        if verbose:
+            self.run_args["verbose"] = None
+            return
+        if "verbose" in self.run_args:
```

Comment:
this is a bit hard to understand. I get that you like the truthy values, but if you keep it like this, lets make sure there are some comments explaining why.

personally I feel like this would be more readable

```python
if verbose:
    ...
else:
    ...
```

Reply:
given this seems to be in a number of places, please either provide a reasoning why this is better then a traditional conditional or change them to a simple if/else.

Reply:
No good reason! Wasn't sure of if there where coding standards against a `if/elif` chains without a default `else` and I wanted to avoid over-indenting with a
```py
if verbose:
    ...
else:
    if "verbose" in dict:  # avoid KeyExp
        ...
```
so i opted for a guard clause instead.

Whole thing can be avoided by simply using `dict.pop`, but I am bad at my job and forgot that method exists!! Fixed in https://github.com/CrayLabs/SmartSim/pull/170/commits/fb759c0a1100c233e79014e682a3e4ca7e5b82fd